NuGet.Config

```diff
@@ -10,6 +10,7 @@
     <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
     <add key="dotnet-msbuild" value="https://dotnet.myget.org/F/msbuild/api/v3/index.json" />
     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
+    <add key="vs-impl" value="https://devdiv.pkgs.visualstudio.com/_packaging/vs-impl/nuget/v3/index.json" />
```

Comment:
@crummel Will we have problems with source-build if we introduce a private NuGet feed to the nuget-client repo?

Reply:
We recently added a workaround for this (for other repos), you should be fine.  Thanks for checking!

Reply:
Historically we have avoided taking a dependency on private feeds. 

We are an OSS project and we should should be mindful of the fact that not all of the NuGet contributors are working for Microsoft. 
The default NuGet development experience shouldn't need anyone to authenticate.